fireapp/lib/pages/Calendar/calendarForm.dart

```diff
@@ -2,19 +2,18 @@
 // ignore: file_names
 // ignore_for_file: no_leading_underscores_for_local_identifiers, use_build_context_synchronously, prefer_typing_uninitialized_variables, file_names
```

Comment:
You shouldn't be touching or editing files in /pages. Only in /presentation (where all the good practice code is, /pages only has legacy, non-functioning views). What do these changes do?

Reply:
This was a legacy file for the old Calendar Form, which was linked from the button. The only meaningful changes were the CalendarFormRoute between lines 12 - 17 to change the navigation from the old form (the code below lines 20ish) to our new form. 

We avoided making any new MVVM code for the OG calendar page just to keep it contained for this PR. It does look like there's been some changes to this old file from lines 20 below from when we were first looking at it. These can be ignored and won't do anything.  If it's okay, happy to get rid of this file/bloat code entirely when we start coding the new Calendar View?